recipes-wpe/wpeframework/wpeframework-plugins-rdk_git.bbappend

```diff
@@ -0,0 +1,4 @@
+PACKAGECONFIG_append = " messenger packager"
```

Comment:
Could you please move  PACKAGECONFIG to the .conf file?

Reply:
done